### PR TITLE
Add a recommendation for GraphQL to be assessed

### PR DESCRIPTION
**Pull request by tmccartan**
A shameless quick add to continue the discussion 

I think we should investigate/ assess whether GraphQL is something we want to use within Gilt. 

I added it as a framework rather than a specific implementation since there is a number of ways to leverage it

### CODE REVIEW

**Comment by tmccartan** [comment]
Didnt save the file, update now. 

Completely open to how we represent it in the repo, I was thinking more something like Server Communication Protocols with REST being the Recommendation and GraphQL being the Assess

**Comment by andreyk0** [comment]
@tmccartan I think in the past we've added items to "assess" category when we had at least one pilot project in production at Gilt. This prevents the doc from becoming a "wish list".

**Comment by tmccartan** [comment]
Understand, but perhaps we should change this line from the recommendations

- **Assess**: Worth exploring with the goal of understanding how it will
  fit at Gilt - does it solve a new problem or improve an existing one?

**Comment by kevinoriordan** [comment]
Yeah I would have thought 

"Assess" = worth exploring
"Trial" = Have pilot projects in production

**Comment by andreyk0** [comment]
Yeah, looks like I got these things mixed up a bit, sorry.
I definitely agree it's worth exploring.

File: recommendations/frameworks/GraphQL.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#GraphQL
+
+##Assess
+
+-[GraphQL] (http://graphql.org/learn/)
+Investigate whether GraphQL is a viable communication protocol within the Gilt Architecture
```
Comment (umatrangolo) [inline recommendations/frameworks/GraphQL.md]: Remove this line. Being in "Assess" already means the same.

Comment (tmccartan) [inline recommendations/frameworks/GraphQL.md]: Updated

**Pull request merged**